.github/workflows/validation.yaml

```diff
@@ -188,7 +188,7 @@ jobs:
         continue-on-error: true
         run: |
           touch output.txt
-          docker extension validate -a -s -i ${{ needs.parse-issue.outputs.repository }} &> output.txt
+          docker extension validate -a -e -s -i ${{ needs.parse-issue.outputs.repository }} &> output.txt
```

Comment:
Nit: for readability, we could use the full flag names maybe

Reply:
done in 0e67806 
See https://github.com/benja-M-1/github-experiment/actions/runs/4293721351/jobs/7481781049#step:12:3